### ISSUE TITLE
Remove generated EXE from repository (umlet-standalone)

### ISSUE THREAD

**Issue by barthel** [umlet/umlet#496]
Commit generated binary resources into source code repository isn't best practice.

Did you ever play around with https://github.com/lukaszlenart/launch4j-maven-plugin to generate the EXE file while maven package phase?

Maybe ```launch4j_project.xml``` should moved into ```src/assembly```, because this file is another descriptor for assemble a distribution.

**Comment by barthel** [comment]
You refers to https://github.com/lukaszlenart/launch4j-maven-plugin/issues/4, right?

It's possible to add a non-active-by-default profile in the ```pom.xml```.
The normal build will not be compromised.

If someone has to generate the EXE file, this profile has to be activate while the Maven build.
```
$ mvn clean install -Pgenerate-exe
```
On the CI, this profile could be always active.

For now I need these libraries too, if I like to generate the EXE via launch4j directly.

**Comment by afdia** [comment]
You're right an optional profile would be a solution, but it's still just a simple exe which wraps an icon and executes the jar file which is external from the exe, so I'm not sure if we would gain anything by replacing the static file with a dynamic creation.

Perhaps I will take a look at it if I find some time, but I would say it's not a high priority issue.

**Comment by barthel** [comment]
> it's not a high priority issue.

I'll build an example for the macOS app (#497) and we will see if it is useful for the EXE too.
